src/interfaces.ts

```diff
@@ -9,6 +9,7 @@ export interface decoratorOptions {
   maxTryCount?: number;
   timeout?: number;
   useAbortController?: boolean;
+  watchStatus?: number;
```

Comment:
I'm not sure that it sounds meaningful to me, let's think of something like `doNotAbortIfStatuses` and make it an array 
```ts
doNotAbortIfStatuses?: number[]
```